widgetry/src/lib.rs

```diff
@@ -24,6 +24,9 @@
 
 //#![warn(missing_docs)]
 
+#[macro_use]
+extern crate log;
```

Comment:
My plan is to eventually do this everywhere, and gradually convert println's to log methods. LMK if you'd like to discuss more before I keep doing this.

Reply:
SGTM for the random `println`s. If you want to modify any of the `timer.note` calls, let's discuss a plan first.